test/jasmine/tests/axes_test.js

```diff
@@ -1264,11 +1264,11 @@ describe('Test axes', function() {
         // way of getting a new clean copy each time.
         function getDefaultAx() {
             return {
+                autorange: true,
                 c2l: Number,
                 type: 'linear',
                 _length: 100,
-                _m: 1,
-                _needsExpand: true
```

Comment:
That's an artefact that of the old range slider + autorange implementation that should've been taken out in https://github.com/plotly/plotly.js/pull/1355/commits/cebc31c72f20cd0e1426e8d9258f9e09e269817a 

My mistake.